firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java

```diff
@@ -50,6 +53,38 @@ public static FirebaseModelDownloader getInstance(@NonNull FirebaseApp app) {
     return app.get(FirebaseModelDownloader.class);
   }
 
+  // Get the downloaded model file based on download type and conditions.
```

Comment:
Usually for multiline comments we use `/*` instead of `//`. 

Also, if you want to get the comment below rendered as a list you would need to use `<ul>` and `<li>`, eg https://github.com/firebase/firebase-android-sdk/blob/master/firebase-database/src/main/java/com/google/firebase/database/Transaction.java#L38

Reply:
Done - thanks.